remotecv/storages/redis_storage.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# thumbor imaging service
+# https://github.com/thumbor/thumbor/wiki
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/mit-license
+# Copyright (c) 2011 globo.com thumbor@googlegroups.com
+
+
+from json import loads
+
+from aioredis import Redis
+from aioredis.sentinel import Sentinel
+from redis import RedisError
+
+
+SINGLE_NODE = "single_node"
+SENTINEL = "sentinel"
+
+
+class Storage:
+    def __init__(self, context):
+        self.context = context
+        self.redis_client = self.get_redis_client()
+
+    async def get_detector_data(self, path):
+        data = await self.redis_client.get(f"thumbor-detector-{path}")
+        if data:
+            return loads(data)
+        return None
+
+    def get_redis_client(self):
+        redis_mode = str(self.context.config.REDIS_QUEUE_MODE).lower()
+
+        if redis_mode == SINGLE_NODE:
+            return self.__redis_single_node_client()
+        if redis_mode == SENTINEL:
+            return self.__redis_sentinel_client()
+
+        raise RedisError(
+            f"REDIS_QUEUE_MODE must be {SINGLE_NODE} or {SENTINEL}"
+        )
+
+    def __redis_single_node_client(self):
+        return Redis(
+            host=self.context.config.REDIS_QUEUE_SERVER_HOST,
+            port=self.context.config.REDIS_QUEUE_SERVER_PORT,
+            db=self.context.config.REDIS_QUEUE_SERVER_DB,
+            password=self.context.config.REDIS_QUEUE_SERVER_PASSWORD,
+        )
+
+    def __redis_sentinel_client(self):
+
+        instances_split = (
+            self.context.config.REDIS_QUEUE_SENTINEL_INSTANCES.split(",")
+        )
+        instances = [
+            tuple(instance.split(":")) for instance in instances_split
```

Comment:
Nit:

```suggestion
            tuple(instance.split(":", 1)) for instance in instances_split
```

as there can only be two elements on those tuples passed as instances.